src/i_video.c

```diff
@@ -1060,6 +1060,8 @@ void I_SetPalette (int palette)
 	    curpane = grnpane;
 	    pane_alpha = 0xff * 125 / 1000;
 	    break;
+	// [crispy] Hexen TODO - add 14-27 palette indexes and extra panes.
+	// But what about coloring formulas?
```

Comment:
Extra panes are easy to implement, there are examples already in the code. Color values could be possibly looked in Slade as it able to generate Hexen palettes very well. Alpha values either should be looked there as well, or will require some higher mathematics. 😦